tripplite_exporter.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import sys
+import time
+from socket import getfqdn
+from typing import Dict, Generator, List, Optional
+
+from prometheus_client.core import GaugeMetricFamily, REGISTRY
+from prometheus_client import start_http_server
+from tripplite.driver import Battery, battery_paths
+
+
+DEFAULT_PORT = 6969
+HOSTNAME = getfqdn()
+LOG = logging.getLogger(__name__)
+
+
+class TrippliteCollector:
```

Comment:
No need to docstring everything but I'd at least add one here.

My main concern is how this code + variable naming would expand if other backends (ex. influxdb) were added. The naming here is too generic to specify that this is for prometheus. My other comments will be along those lines as well.

Reply:
My thought here was a library + entry point for each as they are all would be pretty specific ... Apart from the Battery use logic ...

Would you want a Collector base class and then a Prometheus collector child class?